src/commands/general/tag.rs

```diff
@@ -13,7 +13,7 @@ include!(concat!(env!("OUT_DIR"), "/generated.rs"));
 static TAGS: Lazy<Vec<Tag>> = Lazy::new(|| serde_json::from_str(env!("TAGS")).unwrap());
 
 /// Send a tag
-#[poise::command(slash_command, prefix_command)]
+#[poise::command(slash_command, prefix_command, help_text_fn = help)]
```

Comment:
i'd be fine with this just being a slash command. help text here is just that with more steps + uglier

Reply:
It's useful with slash commands too
It shows the titles of tags

Reply:
> It shows the titles of tags

the slash command already does this on it's own

Reply:
It only shows the ID

Reply:
I made it a lot shorter, I guess

Reply:
I made it a lot shorter and enabled edit tracking properly which I *think* is enough, but annoy me on discord if that's not the case